### PR TITLE
Fix multiline string bug

### LINKED ISSUE #3760: Triple quote multiline strings not supported in interactive shell
I saw #33 and checked it but it seems that nothing has been done for strings that go over two lines, like:

```py
a = """
hello
this is after a newline
"""
```

Is there any reason for this? Or has it just not been implemented yet?
Using branch `main`.

```
Welcome to the magnificent Rust Python 0.1.2 interpreter 😱 🖖
>>>>> """
SyntaxError: Got unexpected string at line 2 column 2

 ^
>>>>>
```

### PR DESCRIPTION
**Pull request by kth496**
Fix #3760 

Related PR: https://github.com/RustPython/RustPython/pull/3874

### CODE REVIEW

File: parser/src/lexer.rs [hunk 1]
```diff
@@ -576,10 +576,16 @@ where
...
+                    return match triple_quoted {
+                        true => Err(LexicalError {
+                            error: LexicalErrorType::Eof,
+                            location: self.get_pos(),
+                        }),
+                        false => Err(LexicalError {
+                            error: LexicalErrorType::StringError,
+                            location: self.get_pos(),
+                        }),
+                    }
```
Comment (Snowapril) [inline parser/src/lexer.rs:588]: We can also do like this
```rust
return Err(LexicalError {
    error: if triple_quoted {
        LexicalErrorType::Eof
    } else {
        LexicalErrorType::StringError
    },
    location: self.get_pos(),
});
```

Comment (Snowapril) [inline parser/src/lexer.rs:588]:
```suggestion
                    return Err(LexicalError { 
                        error: if triple_quoted { 
                            LexicalErrorType::Eof 
                        } else { 
                            LexicalErrorType::StringError 
                        },
                        location: self.get_pos(),
                    });
```

**Comment by youknowone** [comment]
```
Welcome to the magnificent Rust Python 0.1.2 interpreter 😱 🖖
>>>>> """
..... wow
..... """
..... 
'\nwow\n'
```

File: parser/src/lexer.rs (hunk 1 above)
Comment (kth496) [reply parser/src/lexer.rs:588]: @Snowapril Thank you! 👍

**Review by youknowone — APPROVED** [review]
thank you!

**Comment by fanninpm** [comment]
@rayzchen does this fix the problem you were having?

**Comment by rayzchen** [comment]
Yes it does, thanks for closing my issue.

**Pull request merged**